### ISSUE TITLE
How to get it to work

### ISSUE THREAD

**Issue by PaulKushch** [mrjosh/helm-ls#45]
Hello,
Great project!

I am trying to get it to work with `mason`, so I installed `helm_ls`:
![image](https://github.com/mrjosh/helm-ls/assets/5386035/7320510b-6547-4cb7-b088-4e9fe89a6e10)

But it does not work. 

I also tried the guide in readme:
Installed `'towolf/vim-helm'`

Installed `helm_ls`
```
 sudo curl -L https://github.com/mrjosh/helm-ls/releases/download/master/helm_ls_linux_64 --output /usr/local/bin/helm_ls
 sudo chmod +x /usr/local/bin/helm_ls
```

But still nothing happens.

Best Regards

**Comment by hudzy** [comment]
same here, did the exact same process as OP, but helm_ls is not being activated somehow.

**Comment by PaulKushch** [comment]
> same here, did the exact same process as OP, but helm_ls is not being activated somehow.

@hudzy if you use mason then just install with mason, works for me now. Maybe u need yamlls

**Comment by hudzy** [comment]
> > same here, did the exact same process as OP, but helm_ls is not being activated somehow.
> 
> @hudzy if you use mason then just install with mason, works for me now. Maybe u need yamlls

Thanks @PaulKushch , yes the `helm_ls` binaty should be installed via Mason not manually.

I have made `helm_ls` running after going through the error logs. In my case, the issue was due to no `values.yaml` or `values.yml` in the helm chart root dir, so helm_ls errored out.

Any chance @mrjosh could add a feature to customize values file names? Maybe a niche use case but any responses would be highly appreciated.
